### PR TITLE
Update fujifilm

### PR DESCRIPTION
**Pull request by PratyushJayachandran**
fujifilm

**Issue:** closes #

**Similarweb rank:**
  

### Checklist
  - [x] I updated the JSON data in `_data/simple-icons.json`
  - [x] I optimized the icon with SVGO or SVGOMG
  - [x] The SVG `viewbox` is `0 0 24 24`

### Description
SVG: Taken from website header https://www.fujifilm.com/us/themes/custom/fujifilm_com_g2/common/img/fujifilm_corporate_logo.svg
Color: `#FB0020` Taken from the same SVG. Although it can also be green based on Fujifilm packaging since a long time. https://www.fujifilm.com/us/en/social-media/apps

### CODE REVIEW

**Comment by PratyushJayachandran** [comment]
The icon is already in the project but I am not sure if we can add their logo after reading the terms
https://www.fujifilm.com/us/en/terms

File: _data/simple-icons.json [hunk 1]
```diff
@@ -4352,8 +4352,9 @@
         },
         {
             "title": "Fujifilm",
-            "hex": "ED1A3A",
-            "source": "https://upload.wikimedia.org/wikipedia/commons/a/a1/Fujifilm_logo.svg"
+            "hex": "FB0020",
+            "source": "https://www.fujifilm.com/us/themes/custom/fujifilm_com_g2/common/img/fujifilm_corporate_logo.svg",
```
Comment (adamrusted) [inline _data/simple-icons.json:4356]:
```suggestion
            "source": "https://www.fujifilm.com",
```
I'd suggest we just link to the website, if it's available in the header - saves us getting a 404 if/when they change the filename! 🙃

**Review by adamrusted — CHANGES_REQUESTED** [review]
Icon itself looks perfect, just one nit-pick regarding the source. Otherwise, great job @PratyushJayachandran!

**Comment by PratyushJayachandran** [comment]
> Icon itself looks perfect, just one nit-pick regarding the source. Otherwise, great job @PratyushJayachandran!

I usually only add the website but I saw other icons with absolute address so I thought I'll do the same🙈

Will stick to just the website unless they have a branding page.

**Review by adamrusted — APPROVED** [review]
LGTM! Thanks @PratyushJayachandran 🎉

**Pull request merged**